daemon/daemon_unix.go

```diff
@@ -419,6 +419,7 @@ func initBridgeDriver(controller libnetwork.NetworkController, config *Config) e
 		netlabel.DriverMTU:        strconv.Itoa(config.Mtu),
 		bridge.EnableIPMasquerade: strconv.FormatBool(config.Bridge.EnableIPMasq),
 		bridge.EnableICC:          strconv.FormatBool(config.Bridge.InterContainerCommunication),
+		netlabel.EnableIPv6:       strconv.FormatBool(config.Bridge.EnableIPv6),
```

Comment:
@aidanhs apologies on the delayed response. As you know libnetwork doesn't work on driver specific options/labels. Driver options are transparently sent to the drivers/plugins. Whereas libnetwork handles some of the generic labels such as `enable_ipv6` to provide driver independent functionality. In this case, `enable_ipv6` is used to update DNS records. That is the history behind keeping the enable_ipv6 flag as a driver independent flag (but still send it to the driver after libnetwork processing).
Having said that, I think there is an issue for user-defined networks wherein libnetwork doesn't process the this flag. I will confirm.

@mrjana can add more details to this.